configs/anyrtc.json

```diff
@@ -1,10 +1,72 @@
 {
   "index_name": "anyrtc",
   "start_urls": [
-    "https://docs.anyrtc.io/rtc-android/docs/"
+    {
+      "url": "https://docs.anyrtc.io/",
+      "tags": ["all"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
```

Comment:
Please put this item at the item since only the first item matching the page will apply. Since this URL is the most global, only this item will apply and all other item will have no effect.[ Please refer to the documentation for more details](https://docsearch.algolia.com/docs/config-file/#selectors_key-tailor-your-selectors)

Reply:
This point is not resolved. @chudongvip Can you clarify why you marked this as resolved?